packages/components/src/components/contract-card/contract-card-items/contract-card-header.tsx

```diff
@@ -71,12 +71,14 @@ const ContractCardHeader = ({
     const is_mobile = isMobile();
     const is_turbos = isTurbosContract(contract_type);
     const is_lookbacks = isLookBacksContract(contract_type);
+    const is_multipliers = !!multiplier && !is_lookbacks;
```

Comment:
`multiplier` is `number | undefined`. Can't be 0.

Reply:
Why does it need to check if it's lookbacks or not?

Reply:
can we determine it based on `contract_type`?

Reply:
Yeh, we can, but `multiplier` and `is_lookbacks` will still be used in this file